package.json

```diff
@@ -9,7 +9,9 @@
     "bizcharts": "^3.5.6",
     "connected-react-router": "^6.6.1",
     "customize-cra": "^0.9.1",
-    "dotenv": "^7.0.0",
+    "firebase": "^7.6.1",
+    "firebase-admin": "^8.9.0",
+    "firebase-functions": "^3.3.0",
```

Comment:
Are you using these anywhere?

Reply:
Yes I am for the event listeners and the firebase file in the utilities folder